src/com/axis/rtspclient/RTSPClient.as

```diff
@@ -311,7 +311,9 @@ package com.axis.rtspclient {
 
         if (state === STATE_SETUP) {
           /* this is not the case when falling through, e.g. SETUP of first track */
-          if (!/RTP\/AVP\/TCP;unicast;interleaved=/.test(parsed.headers["transport"])) {
+          if (!(/^RTP\/AVP\/TCP;/.test(parsed.headers["transport"]) &&
+        	/unicast;/.test(parsed.headers["transport"]) &&
+        	/interleaved=/.test(parsed.headers["transport"]) )){
```

Comment:
Yes, the semi-colon after `unicast` was what I was referring to. But reading the [RFC](http://www.ietf.org/rfc/rfc2326.txt) now, it seems as what you've done should be enough for what Locomote can support.

The `interleaved` parameter must be set (as you said). And the `Transport` header must have the parameters in this order: `RTP/AVP[/(TCP|UDP)];(unicast|multicast)[; .. parameters]` so there will always be a semicolon after unicast - otherwise Locomote won't be able to play it.

It think it's cleaner to remove the semicolon there though.